guidance/content/regulations/generic_universal.html

```diff
@@ -31,10 +31,10 @@
 <div class="hero">
     <div class="inner-wrap group">
       <div class="primary">
-        <h2 class="hero-header">A platform to read regulations.</h2>
-        <p class="hero-text">eRegulations makes regulations easier to find, read, and understand.</p>
-
-        <p class="read-more-link"><a href="{% url 'about' %}" class="go">About eRegulations</a></p>
+        <h2 class="hero-header">Join our user research for this tool</h2>
+        <p class="hero-text">This is a DSG pilot project to make it easier to research and draft regulations and subregulatory guidance for CMCS work.</p>
+        <p class="hero-text">Try this: go to <a href="/433-112/2016-27844#433-112">Part 433, Section 112</a>, and see the subregulatory guidance in the sidebar, or <a href="diff/433-112/E8-27810/2016-27844?from_version=2016-27844#433-112">see what changed from 2008 to 2017</a>.</p>
```

Comment:
`<a href="/433-112/2016-27844#433-112">` is going to break when deployed. AWS is prefixing our path with `/dev` at the moment. A relative link should work for now, I tested the diff link and it worked.

Reply:
@whytheplatypus Updated!